joomla/joomla-cms
# Delete unnecesary catslug query from mod_related_items

This PR just removes part of sql request which retrieves catslug because it's not used anywhere.

There is a query part that retrieves catslug, but this catslug parameter is never used in this module and doesn't impact anything because later another catslug is created.
## How to test:

Just `print_r($item->catslug);` after [this line](https://github.com/joomla/joomla-cms/pull/8946/files#diff-41447c87f8416eb947248fc54df6cbebL182) and make sure that this parameter is still present even after we remove this part of the sql request.
